pkg/policies/policies.go

```diff
@@ -177,7 +138,12 @@ func (pv *PolicyVerifier) getMaterialPayload(ctx context.Context, m *v12.Attesta
 	// Use the CAS to look for the material
 	var b bytes.Buffer
 	w := bufio.NewWriter(&b)
-	err := pv.cas.Download(ctx, w, m.GetArtifact().GetDigest())
+
+	client, err := pv.getCASClient(pb.CASCredentialsServiceGetRequest_ROLE_DOWNLOADER, m.GetArtifact().GetDigest())
```

Comment:
are you handling when the materials are stored inline?

Reply:
Yes, in the policies logic, it checks if the material is inline, and in that case, it retrieves the content from the `Content` field.